android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java

```diff
@@ -225,6 +236,11 @@ public void setRippleColor(ButtonViewGroup view, Integer rippleColor) {
     view.setRippleColor(rippleColor);
   }
 
+  @ReactProp(name = "exclusive")
+  public void setExclusive(ButtonViewGroup view, Boolean  exclusive) {
```

Comment:
can we just use `boolean` instead of `Boolean` (you can also set default value via `@ReactProp` in case it is missing)